Fix whitespace errors in generate_road_colours.py

There are some in the `generate_colours` function

Which kind of errors? Running `pep8` on the script gives only some warnings, like:

    scripts/generate_road_colours.py:12:1: E302 expected 2 blank lines, found 1
    scripts/generate_road_colours.py:23:80: E501 line too long (112 > 79 characters)
    scripts/generate_road_colours.py:46:1: W293 blank line contains whitespace
    scripts/generate_road_colours.py:73:52: E251 unexpected spaces around keyword / parameter equals

--- a/scripts/generate_road_colours.py
+++ b/scripts/generate_road_colours.py
@@ -43,7 +43,7 @@ def generate_colours(settings, section):
     road_classes = settings['roads']
     colour_divisions = len(road_classes) - 1
     hues = OrderedDict()
-    
+
     min_h = settings['hue'][0]
     max_h = settings['hue'][1]
 

